certified-connectors/Meisterplan/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7710 @@
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ObsTypeResponse"
+            }
+          },
+          "400": {
+            "description": "Bad Request"
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "403": {
+            "description": "Forbidden"
+          },
+          "404": {
+            "description": "Not Found"
+          },
+          "409": {
+            "description": "Conflict"
+          },
+          "429": {
+            "description": "Too Many Requests"
+          }
+        },
+        "security": [
+          {
+            "API-Token": []
+          },
+          {
+            "OAuth": []
+          }
+        ]
+      }
+    },
+    "/v1/obsTypes/{obsTypeId}": {
+      "get": {
+        "tags": [
+          "OBS"
+        ],
+        "summary": "Get an OBS Type by ID",
+        "description": "This endpoint is for OBS types only. To get OBS units, use /obsTypes/<obs-type-id>/obsUnits",
+        "operationId": "GetObsTypeById",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "obsTypeId",
+            "in": "path",
+            "description": "Internal Meisterplan identifier",
+            "required": true,
+            "type": "string",
+            "x-example": "0b9e0e29-8d87-4db8-baa8-1b1a49e354da",
+            "x-ms-summary": "ObsTypeId",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ObsTypeResponse"
+            }
+          },
+          "400": {
+            "description": "Bad Request"
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "403": {
+            "description": "Forbidden"
+          },
+          "404": {
+            "description": "Not Found"
+          },
+          "429": {
+            "description": "Too Many Requests"
+          }
+        },
+        "security": [
+          {
+            "API-Token": []
+          },
+          {
+            "OAuth": []
+          }
+        ]
+      },
+      "delete": {
+        "tags": [
+          "OBS"
+        ],
+        "summary": "Delete an OBS Type by ID",
+        "description": "Delete the OBS type specified by the given ID. If the OBS type with the given ID does not exist, the request will fail.",
+        "operationId": "DeleteObsType",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "obsTypeId",
+            "in": "path",
+            "description": "Internal Meisterplan identifier",
+            "required": true,
+            "type": "string",
+            "x-example": "0b9e0e29-8d87-4db8-baa8-1b1a49e354da",
+            "x-ms-summary": "ObsTypeId",
```

Comment:
Since these are technical terms of our product, we assume that clients from all locales should be familiar with them, and they don't need to be translated.